packages/autorest.python/autorest/codegen/models/base_builder.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         self.api_versions: List[str] = yaml_data["apiVersions"]
         self.added_on: Optional[str] = yaml_data.get("addedOn")
         self.external_docs: Optional[Dict[str, Any]] = yaml_data.get("externalDocs")
+        self.is_lro: bool = yaml_data.get("discriminator") in ("lro", "lropaging")
```

Comment:
we shouldn't have `is_lro` on the base, what we can add is make the LROOperation / LROPagingOperation discriminated types as well / access `builder.yaml_data["discriminator"]` later on

Reply:
Got it. It is only used to avoid duplicated builder for lro so I move it to `RequestBuilderBase`